.storybook/preview.js

```diff
@@ -1,8 +1,6 @@
 import "../src/index.css";
 import "./storybook.css";
-// import { addDecorator } from "@storybook/react";
-// import { withInfo } from "@storybook/addon-info";
-// import { withKnobs } from "@storybook/addon-knobs";
+import { addDecorator } from "@storybook/react";
+import { withKnobs } from "@storybook/addon-knobs";
 
-// addDecorator(withInfo);
-// addDecorator(withKnobs);
+addDecorator(withKnobs);
```

Comment:
i think can remove if not used at all, deprecated liao also -> https://github.com/storybookjs/storybook#deprecated-addons

from what i rmb think they moved most of the addons to `addon-essentials` -> https://storybook.js.org/docs/react/essentials/introduction

Reply:
@simboonlong thanks! yeah i wasn't sure whether is needed so i restore it back ..
but i still need to clean up other stuff..just raising this pr in advance..thanks